### PR TITLE
Adding a new method to the User class check if the user has visited the notification page

### PR DESCRIPTION
**Pull request by jasminechau2**
Creating Notification Class

### CODE REVIEW

File: src/main/java/codeu/model/data/User.java [hunk 1]
```diff
@@ -40,6 +41,7 @@ public User(UUID id, String name, String passwordHash, Instant creation) {
     this.creation = creation;
     aboutMe = "";
     isAdmin = false;
+
```
Comment (lealgamb) [inline src/main/java/codeu/model/data/User.java]: It might be a good idea to set a value for `lastSeenNotificationsTimestamp` in the constructor. Perhaps it can be null that way we know the user is new/hasn't seen any notifications.

File: src/main/java/codeu/controller/NotificationServlet.java [hunk 2]
```diff
@@ -8,6 +8,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.mindrot.jbcrypt.BCrypt;
+import java.util.ArrayList;
```
Comment (lealgamb) [inline src/main/java/codeu/controller/NotificationServlet.java]: nit: these will probably be used later on but it's good to only keep the imports that are needed for this PR

File: src/main/java/codeu/model/data/User.java (hunk 1 above)
Comment (yebrahim) [reply src/main/java/codeu/model/data/User.java]: +1.
You can also add a unit test for it, to make sure it gets initialized to null for a new user.

**Comment by yebrahim** [comment]
How about we change the PR title to reflect the changes? Something about adding the new field to the User class. :)

File: src/main/java/codeu/model/data/User.java (hunk 1 above)
Comment (jasminechau2) [reply src/main/java/codeu/model/data/User.java]: Would it be better to set the time the creation of the user class? Instant has a function now() that sets the time at the current moment because we wouldn't want new users to see past notifications.

Comment (yebrahim) [reply src/main/java/codeu/model/data/User.java]: I think this is actually more accurate, thanks for catching that. Yes we should probably use the current time as value for this field on new users.

File: src/main/java/codeu/controller/NotificationServlet.java [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 import codeu.model.data.User;
 import codeu.model.store.basic.UserStore;
+import codeu.model.data.Event;
```
Comment (yebrahim) [inline src/main/java/codeu/controller/NotificationServlet.java]: I don’t think you need most of these changes. Only the initialization of the new field in User.java.

File: src/main/webapp/WEB-INF/view/notifications.jsp [hunk 4]
```diff
@@ -14,5 +14,6 @@
 <body>
   <%@ include file="navbar.jsp" %>
   <p> Hello User This is your notifcation page</p>
+  <% request.getAttribute("eventsToShow");%>
```
Comment (vocatwk) [inline src/main/webapp/WEB-INF/view/notifications.jsp]: Where do you set the eventsToShow attribute?

File: src/main/java/codeu/controller/NotificationServlet.java [hunk 5]
```diff
@@ -8,9 +8,14 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import org.mindrot.jbcrypt.BCrypt;
+import java.util.ArrayList;
+import java.util.List;
+import java.time.Instant;
 
 public class NotificationServlet extends HttpServlet{
   private UserStore userStore;
+  private Instant timeUserVistedPage;
```
Comment (yebrahim) [inline src/main/java/codeu/controller/NotificationServlet.java]: Why do we need this?
